packages/darklang/languageTools/writtenTypesToProgramTypes.dark

```diff
@@ -1416,5 +1416,7 @@ module Darklang =
         : (ProgramTypes.SourceFile.SourceFile *
```

Comment:
why do we have `parsedFileAsSourceFile` as a separate function, and not just as a part of the `sourceFile` function?

Reply:
not sure :) can probably tidy.